src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift

```diff
@@ -10,9 +10,11 @@ class DataDonationViewController: DynamicTableViewController, DeltaOnboardingVie
 	// MARK: - Init
 
 	init(
-		viewModel: DataDonationViewModelProtocol
+		viewModel: DataDonationViewModelProtocol,
+		largeTitleDisplayMode: UINavigationItem.LargeTitleDisplayMode
```

Comment:
You could define here a default value like `.never`. So only when needed you pass `.always`

Reply:
Sure. Good idea.

Reply:
👌